packages/grafana-data/src/types/index.ts

```diff
@@ -7,7 +7,6 @@ export * from './dashboard';
 export * from './query';
 export * from './annotations';
 export * from './logs';
-export * from './logsVolume';
```

Comment:
@ivanahuckova removing this file from grafana-data breaks our plugins API. If you are planning on removing this and replacing them for others, please follow the procedure of deprecating them in this PR and only removing them in the next major version.

Reply:
Hello @academo! Is this needed as `DataSourceWithLogsVolumeSupport` API had `@internal` tag and we suspected that it might change?  We replaced this with more generic `DataSourceWithSupplementaryQueriesSupport` and implemented change for all internal plugins that were using `DataSourceWithLogsVolumeSupport`.

Reply:
Hi @ivanahuckova I checked the information we have on API usage and we could not find any external plugins using it. so go ahead with merging this PR

Reply:
@ivanahuckova 


- https://github.com/grafana/newrelic-datasource/blob/main/src/datasource/Datasource.ts#L99 
- https://github.com/TencentCloud/tencentcloud-monitor-grafana-app/blob/1b1b240989b27fe111498539cbc109ef8ac97ff2/src/datasource/log-service/LogServiceDataSource.ts#L198

Reply:
The [TencentCloud/](https://github.com/TencentCloud/tencentcloud-monitor-grafana-app/blob/1b1b240989b27fe111498539cbc109ef8ac97ff2/src/datasource/log-service/LogServiceDataSource.ts#L198) is not implementing an API, so there is no issue. 

However [newrelic](https://github.com/grafana/newrelic-datasource/blob/main/src/datasource/Datasource.ts#L99) is implementing and using it (fyi @yesoreyeram). 

Considering this API was marked as `@internal`, would you recommend to:
- keep this PR in this form and let NewRelic data source maintainers know about this
- keep this PR in this form, but add `breaking change` label and let NewRelic data source maintainers know about this
- Change PR to deprecate old API and introduce new one. And then in v10, break the old API
- Something else

Reply:
👍 I brought `DataSourceWithLogsVolumeSupport` back and deprecated it: https://github.com/grafana/grafana/commit/f5be7ad433a062d94fad18d9d08da3435382cac9.